src/intentManager.js

```diff
@@ -209,23 +217,6 @@ define([
         return fields;
     }
 
-    function printTemplate(mug, options) {
-        var widget = widgets.media(mug, options),
-            uploadComplete = widget.handleUploadComplete;
-
-        widget.handleUploadComplete = function (event, data, objectMap) {
```

Comment:
The html files being uploaded are generated by word and it's crashing the flash uploader without a traceback. Apparently https://github.com/dimagi/MediaUploader/pull/9 was not enough. Removing this functionality for now